src/frontend/screens/Exchange/ExchangeSettingsBottomSheet.tsx

```diff
@@ -0,0 +1,210 @@
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+
+const m = defineMessages({
+  save: {
+    id: 'screens.ExchangeSettingsBottomSheet.save',
+    defaultMessage: 'Save',
+  },
+  close: {
+    id: 'screens.ExchangeSettingsBottomSheet.close',
+    defaultMessage: 'Close',
+  },
+  everythingTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingTitle',
+    defaultMessage: 'Exchange Everything',
+  },
+  everythingDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.everythingDesc',
+    defaultMessage: 'Full size photos and audio. Uses more storage.',
+  },
+  previewsTitle: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsTitle',
+    defaultMessage: 'Exchange Previews Only',
+  },
+  previewsDesc: {
+    id: 'screens.ExchangeSettingsBottomSheet.previewsDesc',
+    defaultMessage: 'Reduced smaller size photos. No audio included.',
+  },
+});
+
+export const ExchangeSettingsBottomSheet = () => {
+  const {mutate: setMediaSyncSetting} = useSetMediaSyncSetting();
+  const currentSetting = useGetMediaSyncSetting();
+  const {formatMessage: t} = useIntl();
+  const [selected, setSelected] = useState<MediaSyncSetting>('everything');
+  const {goBack} = useNavigationFromRoot();
+
+  const handleSave = () => {
+    setMediaSyncSetting(selected);
+  };
+
+  React.useEffect(() => {
+    if (currentSetting) setSelected(currentSetting);
+  }, [currentSetting]);
+
+  return (
+    <BottomSheetWrapper>
+      <ScrollView contentContainerStyle={styles.container}>
+        <View style={styles.container}>
+          <OptionCard
+            setting="everything"
+            icon={<OrangeStar width={30} height={30} />}
+            title={t(m.everythingTitle)}
+            description={t(m.everythingDesc)}
+            selected={selected}
+            onSelect={setSelected}
+            testID="EXCHANGE.option-everything"
+          />
+          <OptionCard
+            setting="previews"
+            icon={<GreyLeaf width={30} height={30} />}
+            title={t(m.previewsTitle)}
+            description={t(m.previewsDesc)}
+            selected={selected}
+            onSelect={setSelected}
+            testID="EXCHANGE.option-previews"
+          />
+          <PrimaryButton
+            fullSize
+            style={styles.saveButton}
+            onPress={handleSave}
+            text={t(m.save)}
+          />
+          <SecondaryButton
+            fullSize
+            style={styles.closeButton}
+            text={t(m.close)}
+            onPress={() => goBack()}
+          />
+        </View>
+      </ScrollView>
+    </BottomSheetWrapper>
+  );
+};
+
+const OptionCard = ({
+  setting,
+  icon,
+  title,
+  description,
+  selected,
+  onSelect,
+  testID,
+}: {
+  setting: MediaSyncSetting;
+  icon: React.ReactElement;
+  title: string;
+  description: string;
+  selected: MediaSyncSetting;
+  onSelect: (s: MediaSyncSetting) => void;
+  testID: string;
+}) => {
+  const isSelected = selected === setting;
```

Comment:
can specify `isSelected` as a prop instead of doing the calculation internally:

```suggestion
  isSelected,
  onSelect,
  testID,
}: {
  setting: MediaSyncSetting;
  icon: React.ReactElement;
  title: string;
  description: string;
  isSelected: boolean;
  onSelect: (s: MediaSyncSetting) => void;
  testID: string;
}) => {
```